local-cli/generator-android/templates/src/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +13,7 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
+		android:screenOrientation="portrait"
```

Comment:
Why restrict portrait? RN apps work properly as long as you don't use the dimension constants.